x/tokenomics/keeper/settle_session_accounting_test.go

```diff
@@ -106,11 +107,6 @@ func TestSettleSessionAccounting_ValidAccounting(t *testing.T) {
 	app := apptypes.Application{
 		Address: sample.AccAddress(),
 		Stake:   &appStake,
```

Comment:
We should have `app` staked for `service`.

```suggestion
		Stake:   &appStake,
		ServiceConfigs: []*sharedtypes.ApplicationServiceConfig{
			{
				Service: service,
			},
		},
```